lib/db.js

```diff
@@ -9,6 +9,15 @@ const P = require('./promise')
 const Pool = require('./pool')
 const random = require('./crypto/random')
 
+const REDIS_SESSION_TOKEN_PROPERTIES = [
+  'lastAccessTime', 'uaBrowser', 'uaBrowserVersion',
+  'uaOS', 'uaOSVersion', 'uaDeviceType', 'uaFormFactor'
+]
+
+const REDIS_SESSION_TOKEN_LOCATION_PROPERTIES = [
+  'city', 'state', 'stateCode', 'country', 'countryCode'
+]
```

Comment:
A nit from future me: can you please add some comments here to ensure that future edits to these arrays maintain existing order, add new items at the end, null out placeholders for deleted properties, etc?  It may not be obvious how important the precise order of these lists are to a casual reader.